### PR TITLE
Add hyperlinks to ui-ux-design.md

### PR DESCRIPTION
**Pull request by Ramdoys**
## Changes

Added hyperlinks to captions in the UI and UX Design page. Text change on one of the captions.

## Checks

By submitting your pull request for review, you agree to the following:

- [x] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [x] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [x] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

**Comment by Bianca554** [comment]
Hi, I speak Portuguese, but I'm going to talk about what could help and increase the number of people on Roblox. First, you could put some free items or some items to get on UGC maps, but the UGC maps that are available now are very difficult to make. Second, I think you could give like 15...

**Comment by Bianca554** [comment]
I think everyone would want to play if you gave at least 20 or however much you want and it's also good to lower the Robux stuff a little, it's very expensive like the game called brookeraven and it's too expensive...

File: content/en-us/production/game-design/ui-ux-design.md [hunk 1]
```diff
@@ -50,19 +50,19 @@ In [_Super Striker League_](https://www.roblox.com/games/3360853050/), player in
 <GridContainer numColumns="2">
   <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-6.png" width="100%"/>
-    <figcaption>Contextual buttons in _Super Striker League_</figcaption>
+    <figcaption>Contextual buttons in [_Super Striker League_](https://www.roblox.com/games/3360853050/)</figcaption>
```
Comment (IgnisRBX) [inline content/en-us/production/game-design/ui-ux-design.md:53]:
```suggestion
    <figcaption>Contextual buttons in [Super Striker League](https://www.roblox.com/games/3360853050/)</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 2]
```diff
@@ -37,11 +37,11 @@ Irrelevant elements can cause confusion and distraction. [_Spellbound RPG_](http
 <GridContainer numColumns="2">
   <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-3.png" alt="A close up view of an unselected circular Items button with a brown book icon. The environment in the background is a brown dirt with a patch of snow." width="100%"/>
-    <figcaption>Items prior to selection in *Spellbound RPG*</figcaption>
+    <figcaption>Items prior to selection in [*Spellbound RPG*](https://www.roblox.com/games/6468323505/)</figcaption>
```
Comment (IgnisRBX) [inline content/en-us/production/game-design/ui-ux-design.md:40]:
```suggestion
    <figcaption>Items prior to selection in [Spellbound RPG](https://www.roblox.com/games/6468323505/)</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 3]
```diff
@@ -37,11 +37,11 @@ Irrelevant elements can cause confusion and distraction. [_Spellbound RPG_](http
...
   </figure>
   <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-2.png" alt="A close up view of a selected circular Items button with a brown book icon, along with three additional buttons for skills, equipment, and the players deck. The additional buttons only display when the button is selected." width="90%"/>
-    <figcaption>Items after selection in *Spellbound RPG*</figcaption>
+    <figcaption>Items after selection in [*Spellbound RPG*](https://www.roblox.com/games/6468323505/)</figcaption>
```
Comment (IgnisRBX) [inline content/en-us/production/game-design/ui-ux-design.md:44]:
```suggestion
    <figcaption>Items after selection in [Spellbound RPG](https://www.roblox.com/games/6468323505/)</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 4]
```diff
@@ -83,21 +83,21 @@ In [_Jailbreak_](https://www.roblox.com/games/606849621/)'s season pass UI, a st
 
 <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-8.png" width="60%"/>
-    <figcaption> Season pass UI in _Jailbreak_ </figcaption>
+    <figcaption> Season pass UI in [_Jailbreak_](https://www.roblox.com/games/606849621/)</figcaption>
```
Comment (IgnisRBX) [inline content/en-us/production/game-design/ui-ux-design.md:86]:
```suggestion
    <figcaption> Season pass UI in [Jailbreak](https://www.roblox.com/games/606849621/)</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 5]
```diff
@@ -83,21 +83,21 @@ In [_Jailbreak_](https://www.roblox.com/games/606849621/)'s season pass UI, a st
...
   </figure>
 
 In [_Dragon Adventures,_](https://www.roblox.com/games/3475397644/) visual priority is given to the highest-value coin bundle by increasing its size and separating it with more padding, to distinguish it from smaller bundles.
 
 <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-9.png" width="60%"/>
-    <figcaption> Coin bundles in _Dragon Adventures_</figcaption>
+    <figcaption> Coin bundles in [_Dragon Adventures_](https://www.roblox.com/games/3475397644/)</figcaption>
```
Comment (IgnisRBX) [inline content/en-us/production/game-design/ui-ux-design.md:93]:
```suggestion
    <figcaption> Coin bundles in [Dragon Adventures](https://www.roblox.com/games/3475397644/)</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 6]
```diff
@@ -83,21 +83,21 @@ In [_Jailbreak_](https://www.roblox.com/games/606849621/)'s season pass UI, a st
...
   </figure>
 
 In [_Tower Defense Simulator_](https://www.roblox.com/games/3260590327/), both size and proximity are used to visually group and distinguish Daily Tower Skins from Daily Crates.
 
 <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-10.png" width="60%"/>
-    <figcaption> _Tower Defense Simulator_ shop</figcaption>
+    <figcaption>The shop in [_Tower Defense Simulator_](https://www.roblox.com/games/3260590327/)</figcaption>
```
Comment (IgnisRBX) [inline content/en-us/production/game-design/ui-ux-design.md:100]:
```suggestion
    <figcaption>The shop in [Tower Defense Simulator](https://www.roblox.com/games/3260590327/)</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 7]
```diff
@@ -128,7 +128,7 @@ Headers and titles containing significant high-level information should be large
 
 <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-13.png" width="40%"/>
-    <figcaption> Clothing Shop in *Winds of Fortune*</figcaption>
+    <figcaption> Clothing Shop in [*Winds of Fortune*](https://www.roblox.com/games/7714627632/)</figcaption>
```
Comment (IgnisRBX) [inline content/en-us/production/game-design/ui-ux-design.md:131]:
```suggestion
    <figcaption> Clothing Shop in [Winds of Fortune](https://www.roblox.com/games/7714627632/)</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 8]
```diff
@@ -146,11 +146,11 @@ Given their widespread use, players are typically familiar with such conventions
 <GridContainer numColumns="2">
   <figure>
     <img src="../../assets/game-design/ui-ux-design/ui-ux-design-14.png" width="100%"/>
-    <figcaption>X Close Buttons in *Winds of Fortune*, *Dragon Adventures*, *BotClashSimulator*, and *DOORS*</figcaption>
+    <figcaption>X Close Buttons in [*Winds of Fortune*](https://www.roblox.com/games/7714627632/), [*Dragon Adventures*](https://www.roblox.com/games/3475397644/), [*BotClash Simulator*](https://www.roblox.com/games/9300407930/), and [*DOORS*](https://www.roblox.com/games/6516141723/)</figcaption>
```
Comment (IgnisRBX) [inline content/en-us/production/game-design/ui-ux-design.md:149]:
```suggestion
    <figcaption>X Close Buttons in [Winds of Fortune](https://www.roblox.com/games/7714627632/), [Dragon Adventures](https://www.roblox.com/games/3475397644/), [BotClash Simulator](https://www.roblox.com/games/9300407930/), and [DOORS](https://www.roblox.com/games/6516141723/)</figcaption>
```

File: content/en-us/production/game-design/ui-ux-design.md [hunk 9]
```diff
@@ -146,11 +146,11 @@ Given their widespread use, players are typically familiar with such conventions
...
   </figure>
   <figure>
    <img src="../../assets/game-design/ui-ux-design/ui-ux-design-15.png" width="90%"/>
-    <figcaption>Green "Health" stat consistency in *Arcane Odyssey*</figcaption>
+    <figcaption>Green "Health" stat consistency in [*Arcane Odyssey*](https://www.roblox.com/games/3272915504/)</figcaption>
```
Comment (IgnisRBX) [inline content/en-us/production/game-design/ui-ux-design.md:153]:
```suggestion
    <figcaption>Green "Health" stat consistency in [Arcane Odyssey](https://www.roblox.com/games/3272915504/)</figcaption>
```

**Pull request merged**